src/rust/engine/process_execution/src/remote.rs

```diff
@@ -2151,7 +2244,7 @@ mod tests {
         let op_name = "gimme-foo".to_string();
         mock::execution_server::TestServer::new(mock::execution_server::MockExecution::new(
           op_name.clone(),
-          super::make_execute_request(&execute_request, &None, &None)
+          super::make_execute_request(&execute_request, &None, &None, BTreeMap::new())
```

Comment:
Since specifying platform properties is rare, is it worth "overloading" with a longer name to specify them?

Reply:
We already have two optional fields here I'd want to do that for, so I don't really want to introduce one "overload" :( Maybe one day we'll get default arguments...

Reply:
This... doesn't feel like a great argument for continuing to expand the mostly-not-used arguments. But not a blocker.